src/main.js

```diff
@@ -10,6 +10,8 @@ import { INITIAL_STATE } from './store/state'
 
 import App from './App'
 import VueAnalytics from 'vue-analytics'
+import BootstrapVue from 'bootstrap-vue'
+import 'bootstrap-vue/dist/bootstrap-vue.css'
```

Comment:
How is this synced with the bootstrap version used in molgenis? ( or are these separate and is this not a problem ?)

Reply:
This is only the extra styling needed for bootstrap-vue.
The bootstrap.css is included by molgenis.